### PR TITLE
Write test for removeLeadingSlashes

### PR DESCRIPTION
**Pull request by RossWebFS**

### CODE REVIEW

File: tests/unit/src/utils/removeLeadingSlashes.spec.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import removeLeadingSlashes from "@/utils/remove-leading-slashes/removeLeadingSlashes";
+
+describe("removeLeadingSlashes()", () => {
+  it("should remove slash in the start", () => {
+    expect(removeLeadingSlashes("/about")).toBe("about");
+  });
+  it("should remove all slashes if there are only slashes", () => {
+    expect(removeLeadingSlashes("////")).toBe("");
+  })
+  it("shouldn't remove last slashes", () => {
```
Comment (mhevyk) [inline tests/unit/src/utils/removeLeadingSlashes.spec.ts:10]: Let's use just `should` and then description of the test case

File: tests/unit/src/utils/removeLeadingSlashes.spec.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    expect(removeLeadingSlashes("about/")).toBe("about/")
+    expect(removeLeadingSlashes("about///")).toBe("about///")
+  })
+  it("should return an empty string", () => {
```
Comment (mhevyk) [inline tests/unit/src/utils/removeLeadingSlashes.spec.ts:14]: Try to be more specific, not generic. I suggest using `should return an empty string if we pass empty string as and argument`

**Review by mhevyk — CHANGES_REQUESTED** [review]

File: tests/unit/src/utils/removeLeadingSlashes.spec.ts (hunk 1 above)
Comment (RossWebFS) [reply tests/unit/src/utils/removeLeadingSlashes.spec.ts:10]: Replaced **"shouldn't remove last slashes"** with **"should keep last slashes"**

File: tests/unit/src/utils/removeLeadingSlashes.spec.ts (hunk 2 above)
Comment (RossWebFS) [reply tests/unit/src/utils/removeLeadingSlashes.spec.ts:14]: Fixed

**Review by mhevyk — APPROVED** [review]

**Pull request merged**